.github/workflows/monodocs_build.yml

```
@@ -0,0 +1,52 @@
+name: Docs Build
```

The CI is still [skipping tests](https://github.com/flyteorg/flytekit/actions/runs/7170035243) Were the changes pushed up?

```suggestion
name: Monodocs Build
```

---

weird... one sec